Stripe/STPPaymentCardTextField.m

```diff
@@ -780,6 +783,23 @@ - (void)updateImageForFieldType:(STPCardFieldType)fieldType {
     }
 }
 
+- (NSString *)defaultCVCPlaceholder {
+    if (self.viewModel.brand == STPCardBrandAmex) {
+        return @"CVV";
+    } else {
+        return @"CVC";
+    }
+}
+
+- (void)updateCVCPlaceholder {
+    if (self.cvcPlaceholder) {
+        self.cvcField.placeholder = self.cvcPlaceholder;
+    } else {
+        self.cvcPlaceholder = [self defaultCVCPlaceholder];
+    }
+    self.cvcField.accessibilityLabel = [self defaultCVCPlaceholder];
```

Comment:
I initially thought it'd be fine to just use the default for the accessibility label, but I think you're right – updated to set the accessibility label with the custom placeholder too.